can not install

Dear sir,

please help me. i cannot install  i got error


PS D:\Source\2019\ReactNative\test1> npm install react-native-ping --save   
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: test1@0.0.1
npm ERR! Found: react-native@0.67.4
npm ERR! node_modules/react-native
**npm ERR!   react-native@"0.67.4" from the root project**
npm ERR! 
**npm ERR! Could not resolve dependency:
npm ERR! peer react-native@"^0.41.2 || ^0.57.0" from react-native-ping@1.2.6**
npm ERR! node_modules/react-native-ping
npm ERR!   react-native-ping@"*" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR! 
npm ERR! See C:\Users\Chukiat\AppData\Local\npm-cache\eresolve-report.txt for a full report.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\Chukiat\AppData\Local\npm-cache\_logs\2022-03-23T06_56_36_238Z-debug.log

--- a/package.json
+++ b/package.json
@@ -18,6 +18,6 @@
   "author": "tiancailuohao@gmail.com",
   "license": "MIT",
   "peerDependencies": {
-    "react-native": "^0.41.2 || ^0.57.0"
+    "react-native": "^0.41.2 || ^0.57.0 || ^0.68.0"
   }
 }

